docs/resource/cloud.md

```diff
@@ -39,50 +39,6 @@ The following well-known definitions MUST be used if you set this attribute and
   `/subscriptions/<SUBSCIPTION_GUID>/resourceGroups/<RG>/providers/Microsoft.Web/sites/<FUNCAPP>/functions/<FUNC>`.
   This means that a span attribute MUST be used, as an Azure function app can host multiple functions that would usually share
   a TracerProvider.
-
-`cloud.platform` has the following list of well-known values. If one of them applies, then the respective value MUST be used, otherwise a custom value MAY be used.
```

Comment:
Why was this removed from resource?

Reply:
Hmm good question! I'm looking into why `make table-generation` removed it hmm

Reply:
md file for the new cloud registry is also not generated. 👀